.gitignore

```diff
@@ -0,0 +1 @@
+/README.md
```

Comment:
What's up with this? I'm not understanding why we'd want the readme ignored.

Reply:
Because Github actions overwrites the file and will cause issues, so I think the actual README file should be ignored and instead any edits should be done to the template file.

Reply:
Github actions overwrites it even if it's ignored? That's... weird. How do you feel about moving the mod list to its own markdown file separate from the readme?

Reply:
Yes, github actions adds it with `git add --force README.md`. Moving it to a separate file is an option, though I still think that that file should be ignored by default.

Reply:
Ah, makes sense. I like the schema changes so it's time to merge this. Worrying about what file the list goes in (and maybe github pages integration) can be done later.